lib/src/_client.dart

```diff
@@ -359,6 +359,39 @@ class Client {
     );
   }
 
+  /// Discover terms in the index that match a partial String
+  Future<TermEnumResult> termEnum({
```

Comment:
Nit: Let's call this `termsEnum` instead, as that is the official name of the API. Also, the response should be `TermsEnumResult`.